prep-html-links/contact-us/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home Page</title>
+  </head>
+  <body>
+    <h1>Home page</h1>
+    <div>
+      <a href="contact-us.html">Click here to go to the contact page</a>
+    </div>
+  </body>
+</html>
```

Comment:
This `index.html` should be in `prep-html-links` but not inside of `contact-us/`